contrib/python-bindings/tests/cell_accessor_wrapper.py

```diff
@@ -16,14 +16,33 @@
 import unittest
 from PyDealII.Debug import *
 
-
 class TestCellAccessorWrapper(unittest.TestCase):
 
     def setUp(self):
         self.triangulation = Triangulation('2D')
+        self.dim = 2
```

Comment:
This shouldn't be necessary unless there is a bug in `Triangulation`.

Reply:
Removed.

Reply:
My bad and I misread the code. I thought that you were setting the dimension on the Triangulation.